mon-pix/translations/en.json

```
@@ -487,7 +487,7 @@
         }
       },
       "has-focused-out-of-window": {
-        "certification": "We have detected a page switch.'<br>'Your answer will not be validated.",
+        "certification": "We have detected a page switch. Your answer will not be validated.'<br>'If you have been forced to change pages, please inform your invigilator and answer the question.",
```

J'aurais pas mis de 's' à 'pages' mais c'est peut-être validé!?
```suggestion
        "certification": "We have detected a page switch. Your answer will not be validated.'<br>'If you have been forced to change page, please inform your invigilator and answer the question.",
```

---

c'est la trad officielle